pkg/scheduler/framework/plugins/interpodaffinity/filtering_test.go

```diff
@@ -949,16 +958,19 @@ func TestRequiredAffinityMultipleNodes(t *testing.T) {
 				[]runtime.Object{
 					&v1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: "NS1"}},
 				})
+			state := framework.NewCycleState()
+			_, preFilterStatus := p.(framework.PreFilterPlugin).PreFilter(ctx, state, test.pod)
+			if preFilterStatus.Code() != test.wantPreFilterStatus.Code() {
```

Comment:
ditto, use cmp.Diff as noted in the other comment, for both the PreFilter and Filter statues.